packages/angular_devkit/build_webpack/src/webpack-dev-server/index.ts

```diff
@@ -68,11 +73,22 @@ export class DevServerBuilder implements Builder<WebpackDevServerBuilderSchema>
         },
       );
 
+      // Node 8 has a keepAliveTimeout bug which doesn't respect active connections.
+      // Connections will end after ~5 seconds (arbitrary), often not letting the full download
+      // of large pieces of content, such as a vendor javascript file.  This results in browsers
+      // throwing a "net::ERR_CONTENT_LENGTH_MISMATCH" error.
+      // https://github.com/angular/angular-cli/issues/7197
+      // https://github.com/nodejs/node/issues/13391
+      // https://github.com/nodejs/node/commit/2cb6f2b281eb96a7abe16d58af6ebc9ce23d2e96
+      if (/^v8.\d.\d+$/.test(process.version)) {
```

Comment:
We should probably remove this now.  This was corrected in 8.8.  I think we can comfortably tell people to update to the LTS version (8.11) instead of maintaining this.

Reply:
SGTM, removed.